pmd-java/src/main/resources/category/java/design.xml

```diff
@@ -1289,14 +1289,20 @@ public void foo() throws Exception {
         <description>
 <![CDATA[
 Look for ternary operators with the form `condition ? literalBoolean : foo`
-or `condition ? foo : literalBoolean`.
+or `condition ? foo : literalBoolean`
+or `condition ? literalBoolean1 : literalBoolean2`.
```

Comment:
TBH this last one is a special case of the former two. We don't need to list it.